docs/cli/installation.mdx

```diff
@@ -2,6 +2,10 @@
 title: "Installation"
 ---
 <Info>Missing an installation? submit a <a href="https://github.com/keephq/keep/issues/new?assignees=&labels=&projects=&template=use_case.md&title=">new installation</a>  request and we will add it as soon as we can.</Info>
+
+<Info>
+The installation is possible only with Python version 3.11
```

Comment:
Would be cooler to use "positive" sentiment and give some explanation on why (because pyarrow doesn't support python 3.12)

Also - can add some "contact us if you need Keep CLI on other versions)